src/foundry/client-esm/canvas/tokens/ring-config.d.mts

```diff
@@ -72,8 +75,116 @@
  * CONFIG.Token.ring.debugColorBands = true;
  * ```
  */
-declare class TokenRingConfig {}
+declare class TokenRingConfig {
+  #TokenRingConfig: true;
+
+  constructor();
+
+  /**
+   * Core token rings used in Foundry VTT.
+   * Each key is a string identifier for a ring, and the value is an object containing the ring's data.
+   * This object is frozen to prevent any modifications.
+   */
+  static CORE_TOKEN_RINGS: TokenRingConfig.CoreTokenRings;
+
+  /**
+   * Core token rings fit modes used in Foundry VTT.
+   */
+  static CORE_TOKEN_RINGS_FIT_MODES: TokenRingConfig.CoreTokenRingsFitModes;
+
+  /** Register the token ring config and initialize it */
+  static initialize(): void;
+
+  /** Register game settings used by the Token Ring */
+  static registerSettings(): void;
+
+  /** A mapping of token subject paths where modules or systems have configured subject images. */
+  subjectPaths: Record<string, string>;
+
+  /**
+   * All color bands visual debug flag.
+   * @defaultValue `false`
+   */
+  debugColorBands: boolean;
+
+  /**
+   * Get the current ring class.
+   */
+  get ringClass(): typeof TokenRing;
+  set ringClass(value: typeof TokenRing);
```

Comment:
No need to change this but one cool thing about unidirectionally getters/setters is that you only have to put the type in one spot.

```ts
  get ringClass(): typeof TokenRing;
  set ringClass(value);
```
As in this'd work.